tests/utils/stylable-proxy.ts

```diff
@@ -0,0 +1,21 @@
+import {create} from '../../src/runtime';
```

Comment:
Please create a function compatible to the create function in runtime.ts and export it in order to use it in the runtime test suite.

Reply:
you mean just wrapping what I export as a function?

Reply:
you mean just wrapping what I export as a function?